gettsim/transfers/rente.py

```diff
@@ -0,0 +1,340 @@
+from gettsim.piecewise_functions import piecewise_polynomial
+from gettsim.typing import BoolSeries
+from gettsim.typing import FloatSeries
+from gettsim.typing import IntSeries
+
+
+def gesamte_rente_m(priv_rente_m: FloatSeries, ges_rente_m: FloatSeries) -> FloatSeries:
+    """Calculate total pension as sum of private and public pension.
+
+    Parameters
+    ----------
+    priv_rente_m
+        See basic input variable :ref:`priv_rente_m <priv_rente_m>`.
+    ges_rente_m
+        See :func:`ges_rente_m`.
+
+    Returns
+    -------
+
+    """
+    out = priv_rente_m + ges_rente_m
+    return out
+
+
+def ges_rente_m(
```

Comment:
Another open question is how to name public retirement earnings.

In line with the naming of other parts of the code, it would be `ges_rente_m`. The problem here is that there also exists the function `gesamte_rente_m` which it might be confused with.

Reply:
An alternative could be `staatl_rente_m` or `gesetzl_rente_m` (but both would be inconsistent to e.g. `ges_rentenv`, `ges_krankenv`

Reply:
At a minimum, make `gesamte_rente_m` private, if someone really wants it, she can just replicate summing the private and public pensions.

This is opening a can of worms (and not for this PR), but we are dealing with this in an overly simplistic manner. E.g., we cannot simply merge all private pension payments (Betriebsrente, private Rentenversicherung pre-2004, Riester, Rürup, whatever) and apply health insurance contributions to it: https://www.test.de/Steuern-und-Sozialabgaben-Diese-Abgaben-zahlen-Sie-auf-Ihre-Rente-5542858-0/

Reply:
> An alternative could be `staatl_rente_m` or `gesetzl_rente_m` (but both would be inconsistent to e.g. `ges_rentenv`, `ges_krankenv`)

Sorry, did not see this in time -- the problem is the `gesamte`, not this here. `staatl` would be wrong in a legal sense.

Reply:
It is a convenient shorthand in the code but it is not central for users to have access to it.

Reply:
I would say it is relevant for every user who is interested in any of these transfers and would like to find out how the inputs are calculated.

Would you propose to remove ``gesamte_rente_m`` and have ``priv_rente_m`` and ``ges_rente_m`` as separate arguments for the calculation of these functions instead? Otherwise, I don't think we should make the function private.

Maybe we can just rename it instead: `summe_ges_priv_rente_m`?

Reply:
That is a very good suggestion!!! Explicit rather than implicit.

And yes, in the medium run I would think that we just use its parts — it will just be too simplistic to be useful in many places.